fast64_internal/utility.py

```diff
@@ -991,4 +1162,19 @@ def read16bitRGBA(data):
 	b = bitMask(data,  1, 5) / ((2**5) - 1)
 	a = bitMask(data,  0, 1) / ((2**1) - 1)
 
-	return [r,g,b,a]
\ No newline at end of file
+	return [r,g,b,a]
+
+def join_c_args(args: 'list[str]'):
+	return ', '.join(args)
+
+def translate_xyz_to_xzy(translate: mathutils.Vector):
+	xzy_translate = translate.xzy
+	xzy_translate[2] = -xzy_translate[2]
+	return xzy_translate
```

Comment:
```suggestion
	return transform_mtx_blender_to_n64() @ xzy_translate
```

Reply:
lmao all my matrix math is very suspicious now.

just pushed this changed and renamed it to `translate_blender_to_n64`